### PR TITLE
Do not emit error in _getSharedString function

### PR DESCRIPTION
**Pull request by kikill95**
I am not sure why do we need this error emit

The reason why I've faced with this issue is that I tried to process several big xlsx (and xls) files. For my case big file was with 440k of rows.

When I'd removed this error emit files processed correctly

I am not sure is it a proper fix or maybe this is only a wrong workaround, maybe @DaSpawn you will say more about this

### CODE REVIEW

**Comment by DaSpawn** [comment]
Interesting issue, your sheet should never reference a shared string that does not exist, does your sheet have missing information/fields?  How was the source excel sheet created?

I added this error as when developing there was an elements not closed in a particular badly formated xls xml so the fields were not being read properly.  I wonder if this is the same issue.  Eliminating the error is more problematic as you would not know your sheet is potentially missing a lot of information (the shared strings is critical to sheet content). 

this is also an issue if your sheet has the shared strings stored after the xml in the xls archive, so that requires processing the sheet twice, once to obtain the shared strings and a second time to actually emit data, so those particular sheets can not be streamed without storing first

bottom line need to know if your resulting emitted data is complete/correct;  thank you for the fix but unfrortunately not the best way to do.  How were you handling the emitted errors originally?

**Comment by kikill95** [comment]
Thank you for a reply

Our software should work crossplatform and wrong sheets as well and "ignore" them like excelMS and LibreOffice do

These files are huge, but I don't see anything wrong
Also, I don't know how they were created =(

I agreed that eliminating this error isn't the best way, however, maybe we can make "soft" parser or options like some csv or xls parsers do - they provide "silent" option and if this is set to `true` - they ignore such things and insert empty strings in this cells

Thank you for your awesome package, this is really great

I will close this PR, maybe I will find another way and I will reopen this
Or I will make it work for our project with this "silent" mode by default

**Comment by kikill95** [comment]
Also, one more notice is that I know that person, who provided me these docs, convert one type to another manually with file manager, I don't know whether it can be this issue, but just an additional information
